### DISCUSSION TITLE
求助能否设置使用手势打开已划选的链接

### DISCUSSION THREAD

**Discussion by xmbai9507** [shuax/MouseInc.Settings#290] [Q&A] [answered]
在火狐可以直接划选拖拽链接以打开, 最近换到edge, 发现没有这个功能, 请问能用鼠标手势进行一番设置吗, 感谢.

**Comment by shuax** [comment, marked as answer]
做不了。

**Reply by xmbai9507** [reply]
我发现edge其实有提供快捷键ctrl+shift+L, 所以解决了.

**Reply by Faith-Mian** [reply]
> 我发现edge其实有提供快捷键ctrl+shift+L, 所以解决了.

打开剪贴板中的网址是吗？这个快捷键并不顺手，edge别的细节也并不完善，一直没太用。不如百分浏览器自带的默认鼠标手势右下就好，或者右击顶栏的加号。

**Comment by 6Tom** [comment]
能做到，曲线救国，我是这样用的，ctrl+cc不是有快捷菜单么，新建一项，名称命名为“网址(&u)”，修改RAW的代码为这
```
[
	["Execute", "%appdir%\\\\URL.bat"]
]
```
然后在软件所在目录建一个bat，名为“URL.bat”，里面贴上这代码
```
for /f "usebackq delims=" %%a in (`mshta "javascript:var s=clipboardData.getData('text');if(s)new ActiveXObject('Scripting.FileSystemObject').GetStandardStream(1).Write(s);close();"`) do call start "msedge.exe" "%%a" & exit
```

之后想要打开什么网址，选中，按下ctrl+ccu就能打开了

而且环境不局限于浏览器内，系统里面只要能调出mouseinc快捷菜单的位置都可以使用

而且还可以打开资源管理器路径，支持复制的路径中含windows环境变量

**Comment by gbsix** [comment]
先设置一个在浏览器中打开的手势，然后在任意页面复制链接，用手势就可以打开了：
`[ [ "Execute", "D:\\centbrowser_portable\\chrome.exe %clipboard%" ] ] `
